index.html

```diff
@@ -24,9 +24,388 @@ <h1>Logo</h1>
+
+            <section class="what-we-do--section">
+                <h1 class="section-header">
+                    Explore the world's top curated courses in one simple place.
+                </h1>
+
+                <div class="what-we-do--wrapper">
+                    <div class="what-we-do">
+                        <div class="img--wrapper">
+                            <img src="" alt="">
+                        </div>
+
+                        <h2>Explore crowd-sourced courses</h2>
+                        
+                        <p>The programming world changes every couple of seconds. Whether it’s coding, IT, or UX design, access top curated courses in one simple place to stay ahead of the curve.</p>
+                    </div>
+
+                    <div class="what-we-do">
+                        <div class="img--wrapper">
+                            <img src="" alt="">
+                        </div>
+
+                        <h2>Solve a programming query</h2>
+
+                        <p>Even the most experienced coders don’t have all the answers. Find relevant courses, articles, and resources to help you solve any programming query.</p>
+                    </div>
+
+                    <div class="what-we-do">
+                        <div class="img--wrapper">
+                            <img src="" alt="">
+                        </div>
+
+                        <h2>Level up your tech skills</h2>
+
+                        <p>Brush up your knowledge of AI, blockchain, Python, and other in-demand tech skills through any of our tutorials, bootcamps, reviews, and courses.</p>
+                    </div>
+                </div>
+
+                <button class="learn-more--btn">Learn More</button>
+            </section>
+
+            <section class="promotions--section">
+                <div class="our-benefits">
+                    <p>Hackr.io helps millions of global users expand their programming education to reach new career heights. Do you have an offering for our eager tech audience? We’re always looking for new partners to bring more value to our audience.</p>
+                </div>
+
+                <div class="sponsors--wrapper">
+                    <div class="img-container">
+                        <img src="" alt="">
+                    </div>
+
+                    <div class="img-container">
+                        <img src="" alt="">
+                    </div>
+
+                    <div class="img-container">
+                        <img src="" alt="">
+                    </div>
+
+                    <div class="img-container">
+                        <img src="" alt="">
+                    </div>
+
+                    <div class="img-container">
+                        <img src="" alt="">
+                    </div>
+                </div>
+
+                <div class="testimonies--wrapper">
+                    <div class="slider-btn--wrapper">
```

Comment:
The image between the the two buttons should be removed because we are placing the buttons to position absolute and there are individual images for the individual comments and also on desktop view, there are two images per time...